keps/sig-network/1669-proxy-terminating-endpoints/README.md

```diff
@@ -185,6 +185,7 @@ All existing E2E tests for Services should continue to pass.
 
 * E2E tests are in place, exercising all permutations of internalTrafficPolicy and externalTrafficPolicy.
 * Metrics to publish how many Services/Endpoints are routing traffic to terminating endpoints.
+* Rollback testing (manual or automated)
```

Comment:
Sorry - can't comment on the untouched lines - so commenting here.

L228: were those tests added?

L250: were those metrics added?

L362: Can you fill this in? [I guess it's probably as simple as "update workloads to properly set readiness probe"]

Reply:
Also - please adjust the Testing section to the new format.

Reply:
> L228: were those tests added?

Yes, some examples:
https://github.com/kubernetes/kubernetes/blob/d436f5d0b7eb87f78eb31c12466e2591c24eef59/pkg/proxy/iptables/proxier_test.go#L6114 
https://github.com/kubernetes/kubernetes/blob/6e9845f766e4d34620835aaa1e5f864211471a50/pkg/proxy/ipvs/proxier_test.go#L5493

> L250: were those metrics added?

Yes, this should be accomplished with `sync_proxy_rules_no_local_endpoints_total` on a per-node basis.

Will include these links in the PRR and use the new format.

Reply:
Updated, PTAL

Reply:
> Yes, some examples:
https://github.com/kubernetes/kubernetes/blob/d436f5d0b7eb87f78eb31c12466e2591c24eef59/pkg/proxy/iptables/proxier_test.go#L6114
https://github.com/kubernetes/kubernetes/blob/6e9845f766e4d34620835aaa1e5f864211471a50/pkg/proxy/ipvs/proxier_test.go#L5493

Can you please update that in the PR?

Reply:
Yeah I already included them in the Test Plan section using the new format: https://github.com/kubernetes/enhancements/blob/6ddd543896cf6c201352ba302f4354c0e89e8165/keps/sig-network/1669-proxy-terminating-endpoints/README.md#unit-tests

Reply:
Heh...
Can you just add here sth like "(linked in Test Plan section)".
?

Reply:
Sure, updated